doc/ref_cert/RC1/chapters/chapter03.md

```diff
@@ -724,7 +724,15 @@ At the time of writing, no KPI is defined in CNTT chapters which would have
 asked for an update of the default SLA proposed in
 [Functest Benchmarking CNTT](https://git.opnfv.org/functest/tree/docker/benchmarking-cntt/testcases.yaml?h=stable%2Fjerma)
 
-On top of this dataplane benchmarking described in VMTP & Shaker, we need to integrate testing as described in [ETSI GS NFV-TST 009: Specification of Networking Benchmarks and Measurement Methods for NFVI](https://www.etsi.org/deliver/etsi_gs/NFV-TST/001_099/009/03.01.01_60/gs_NFV-TST009v030101p.pdf). This type of testing is better suited to measure the networking capabilities of a compute node. The [rapid scripts](https://wiki.opnfv.org/display/SAM/Rapid+scripting) in conjunction with the [PROX tool](https://wiki.opnfv.org/pages/viewpage.action?pageId=12387840) offers an open source implementation for this type of testing.
+Industry Standard-based Benchmarking Projects
+
+In addition to the dataplane testing described in Functest, two projects have integrated industry standard benchmarking test as described in [ETSI GS NFV-TST 009: Specification of Networking Benchmarks and Measurement Methods for NFVI](https://www.etsi.org/deliver/etsi_gs/NFV-TST/001_099/009/03.04.01_60/gs_NFV-TST009v030401p.pdf), and other IETF standards as well. 
+- [SampleVNF](https://wiki.anuket.io/display/HOME/Sample+VNF)
+- [ViNePERF](https://wiki.anuket.io/display/HOME/ViNePERF)
+
+This type of testing is better suited to measure the networking capabilities of a compute node. The [rapid scripts](https://wiki.opnfv.org/display/SAM/Rapid+scripting) in conjunction with the [PROX tool](https://wiki.opnfv.org/pages/viewpage.action?pageId=12387840) offers an open source implementation for this type of testing.
```

Comment:
Should these Wiki pages be moved over to wiki.anuket.io?

Reply:
Yes, but they have not been moved, I guess this is a resource issue. The links still work, and in a future commit we should update the status of SampleVNF.